builtin/providers/azurerm/config.go

```diff
@@ -99,6 +100,11 @@ type ArmClient struct {
 	serviceBusSubscriptionsClient servicebus.SubscriptionsClient
 
 	keyVaultClient keyvault.VaultsClient
+
+	sqlDatabasesClient               sql.DatabasesClient
+	sqlElasticPoolsClient            sql.ElasticPoolsClient
+	sqlRecommendedElasticPoolsClient sql.RecommendedElasticPoolsClient
+	sqlServersClient                 sql.ServersClient
```

Comment:
Given we're currently using the Riviera SDK for this functionality, this client isn't used. As such - is it possible to remove it for the moment?